tests/e2e_test.go

```diff
@@ -44,3 +44,7 @@ var _ = ginkgo.AfterSuite(testUtils.TearDownNetwork)
 var _ = ginkgo.Describe("[Teleporter integration tests]", func() {
 	ginkgo.It("Send a message from Subnet A to Subnet B", BasicOneWaySend)
 })
+
+var _ = ginkgo.Describe("[Native Token Bridge integration tests]", func() {
+	ginkgo.It("Send native tokens from subnet A to B and back", NativeTokenBridge)
```

Comment:
Let's have this as an `It` spec in the `Teleporter integration tests` `Describe` suite